### DISCUSSION TITLE
Unable to send event to actor

### DISCUSSION THREAD

**Discussion by JoshuaAmaju** [statelyai/xstate#1691] [Q&A] [answered]
I'm getting `Unable to send event to child 'email' from service 'address'`. Even though the actor `email` is a child of `address`.

```typescript
{
      id: "address",
      initial: "idle",
      entry: assign(() => {
        const actors = {};

        for (const key in obj) {
          actors[key] = spawn(actor, key);
        }

        return { actors };
      }),
      states: {
        idle: {
          on: {
            "*": {
              actions: send((_, e) => e, {
                to: (_, { name }) => name,
              }),
            },
          },
        },
```

**Comment by Andarist** [comment, marked as answer]
Please always try to share a repro case in a runnable form - either by providing a git repository to clone or a codesandbox. OSS maintainers usually can't afford the time to set up the repro, even if exact steps are given.

**Reply by JoshuaAmaju** [reply]
Sorry about that, [here it is](https://codesandbox.io/s/billowing-flower-hgpm5?file=/src/machine.ts)

**Reply by JoshuaAmaju** [reply]
I've also noticed that the same code works fine on codesandbox, but doesn't work locally.

**Reply by davidkpiano** [reply]
> I've also noticed that the same code works fine on codesandbox, but doesn't work locally.

Make sure you have the same versions of `xstate` and `@xstate/react`. You can also download the `.zip` of the repository for the CodeSandbox example, run it locally, and compare it to your local version.

**Reply by JoshuaAmaju** [reply]
This is an interesting situation, did as you suggested and it worked fine locally. The initial project I had the issue with used parcel instead of react-scripts. Why should that cause any difference in behavior?

**Reply by JoshuaAmaju** [reply]
Noticed a difference in the output result after spawning an actor. It seems the codesanbox project spawned the actors correctly, but the one I had locally didn't.
